lib/winrm/http/response_handler.rb

```diff
@@ -61,31 +62,50 @@ def raise_if_auth_error
     def raise_if_wsman_fault
       soap_errors = REXML::XPath.match(
         response_xml,
-        "//#{WinRM::WSMV::SOAP::NS_SOAP_ENV}:Body/#{WinRM::WSMV::SOAP::NS_SOAP_ENV}:Fault/*")
+        "//*[local-name() = 'Envelope']/*[local-name() = 'Body']/*[local-name() = 'Fault']/*")
```

Comment:
Nice usage of `local-name()`! Yay xpath??

Reply:
Yeah, I wish there was a better way of doing that - i.e. resolve the namespace by URI dynamically - but that would require a better XML parser, and is probably overkill here... 

Brought back some bad memories of mid-2000 XPath excursions from a former life!